applications/SubsCrypt.md

```diff
@@ -0,0 +1,185 @@
+
+## Team :busts_in_silhouette:
+
+### Team members
+* Name of team leader
+    Saber Zafarpoor (Product manager & Blockchain Designer)
+    Hadi Esna (CTO & Blockchain Designer)
+* Names of team members	
+    Ahmad Salimi(Software Engineer)
+    Alireza Ilami(Data Analyst)
+    Sepehr Kianian(Front-End developer & Blockchain developer)
+    Amir Sadra Abdolahi(Front-End developer & Blockchain developer)
+    Yasaman Shad(Graphic Designer & UX)
+    ...(we will hire blockchain researchers and UI developers to enhance the quality)
+
+
+### Contact
+* **Contact Name:** Saber Zafarpoor
+* **Contact Email:** szafarpoor@ce.sharif.edu
+* https://oxydev.ir
+
+
+### Team's experience
+**OxyDev** consists of young enthusiast students/developers who graduated or undergraduates at #1 university of Iran (Sharif University of Technology) trying to make web3 easier to use and integrate into real-world services. We have developed several Defi projects in Tron and Ethereum Ecosystem and expanded our team to make venturous leaps to play our roles in making a more user-friendly world in the next generation of Web(Web3). We have done several projects of web2 that make our team aware of the pros and cons of web2. 
+
+### Team Code Repos
+* https://github.com/oxydev
+* Our previous projects can not be publicly announced(NDA)
+
+### Team LinkedIn Profiles
+* https://www.linkedin.com/in/szafarpoor/
+* https://www.linkedin.com/in/ahmadsalimi/
+* https://www.linkedin.com/in/hadiesnaashari/
+* https://www.linkedin.com/in/alirezailami/
+
+## Development Roadmap :nut_and_bolt: 
+We will divide our project into four milestones. For the first milestone, we will implement our smart contract can be deployed to a substrate chain using ink! Which is a smart contract substrate pallet. The second and third milestones will be implemented in parallel. The last milestone will be connecting the components to work seamlessly together.
+
+**There will be three ways to connect to our smart contract:**
+1. Using a Polkadot{.js} wallet to interact with the contract(state of the contract can be modified in this scenario)
+2. Using our RestAPI backend or third party library to retrieve data from the contract (state of the contract can not be modified in this scenario)
+3. Using any customized client to interact with the contract
+
+**Auth mechanism :** 
+We have designed a secure and user-friendly auth mechanism that users and providers can easily use it.
+
+Users have to choose a pair of token and passphrase, which we recommend having a common token(more than 16 chars) in every provider that they subscribe but have a different passphrase(can be small). Users will submit sha256(token+passphrase) to the contract, and whenever they want to authenticate themselves, they have to provide these token and passphrase in a view function( which is not a transaction, so it's not on-chain and free).
```

Comment:
Since providing the sha256 hash is not a transaction, this is just a way for external apps to authenticate without interacting with the blockchain?

Reply:
Yeah, we have decided to make our contract usable for all kind of local businesses(even those without websites or those with WordPress), so we had to implement an authentication service that can be easily used by developers and also on the other side users can log in to their services without a wallet(on their smart tv or other gadgets) using a pair of token and passphrase.